From 130a5b0517ff53ce0369414b15e296b2a7195da5 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 18 Jan 2023 23:45:18 +0000
Subject: [PATCH] gnu: cl-croatoan: Update to 0.0.1-7.622a590.

* gnu/packages/lisp-xyz.scm (sbcl-croatoan): Update to 0.0.1-7.622a590.
  [arguments]{asd-systems} Add croatoan, croatoan-ncurses and ansi-escape.
  [arguments]{phases}(fix-paths): Use search-input-file procedure.
---
 gnu/packages/lisp-xyz.scm | 15 +++++++--------
 1 file changed, 7 insertions(+), 8 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 39fe5c3f76..947e7fa6ed 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -17121,8 +17121,8 @@ (define-public cl-tooter
   (sbcl-package->cl-source-package sbcl-tooter))
 
 (define-public sbcl-croatoan
-  (let ((commit "35c92fcc6c4458339c7e27bdf16f7fa908d95eb8")
-        (revision "6"))
+  (let ((commit "622a5902c38b45248bf843e66158a2ee19468018")
+        (revision "7"))
     (package
       (name "sbcl-croatoan")
       (version (git-version "0.0.1" revision commit))
@@ -17134,18 +17134,17 @@ (define-public sbcl-croatoan
                (commit commit)))
          (file-name (git-file-name "cl-croatoan" version))
          (sha256
-          (base32 "0v1lrdjd29krjb6pr0mql9hjj5c2lv760xwpr17dbsw263vx29xz"))))
+          (base32 "015xq5bbmv05bnl9hynw0bc7c6dv49hak72f5h45ry8zmpxx5cr5"))))
       (build-system asdf-build-system/sbcl)
       (arguments
-       '(#:phases
+       '(#:asd-systems '("croatoan" "croatoan-ncurses" "ansi-escape")
+         #:phases
          (modify-phases %standard-phases
            (add-after 'unpack 'fix-paths
              (lambda* (#:key inputs #:allow-other-keys)
                (substitute* "ncurses/ncurses.lisp"
-                 (("libncursesw" all)
-                  (string-append (assoc-ref inputs "ncurses")
-                                 "/lib/"
-                                 all))))))))
+                 (("libncursesw.so")
+                  (search-input-file inputs "/lib/libncursesw.so"))))))))
       (inputs
        (list ncurses
              sbcl-cffi
-- 
2.38.1

